package.json

```diff
@@ -64,7 +64,9 @@
     "whatwg-fetch": "^2.0.3"
   },
   "scripts": {
-    "start": "./node_modules/devloop/bin/main.js",
```

Comment:
Sorry I missed that during the review but it is better to use the local version of devloop. Why did you change it? I think it should be reverted.